gradlew

```diff
@@ -163,10 +163,18 @@ fi
 
 CLASSPATH=$GRADLE_WRAPPER_JAR
 
-# Don't fork a daemon mode on initial run that generates local defaults.
-GRADLE_DAEMON_CTRL=
+# Generate gradle.properties if they don't exist
 if [ ! -e "$APP_HOME/gradle.properties" ]; then
-    GRADLE_DAEMON_CTRL=--no-daemon
+    echo "Populating gradle.properties from gradle.properties.template."
+    # Do the copy first. If the next steps fail, the gradle.properties will still be usable
+    cp gradle.properties.template gradle.properties
```

Comment:
```suggestion
    cp gradle.properties.template "$APP_HOME/gradle.properties"
```

Also, could the `gradle.properties.template` file be inside `gradle/` folder to avoid littering the root git folder?